recipes/seqan3/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+install -d $PREFIX/include
+install -d $PREFIX/share/cmake
+install -d $PREFIX/share/doc
```

Comment:
The documentation must be build from source (uses `doxygen` that generates html files, not to sure how to add `doxygen` generated documentation to  conda packages)

documentation can be build like this:

```terminal
/my/path/to/seqan3

mkdir build_doc
cd build_doc
cmake /my/path/to/seqan3/test/documentation
make doc_usr
```

in `build_doc/doc_usr/html/` is the complete html documentation.

Reply:
Can you access it with `man`? If not, might be best just to leave it out of this package as it's available online

Reply:
We can also build `man` pages. Just tried it out.

For that we need to add `GENERATE_MAN = YES` to the generated Doxyfile `build_doc/doc_usr/Doxyfile`. We might need to add a `OPTION` in `cmake` to make this work.